server/src/boot/defaultSettings.ts

```diff
@@ -7,5 +7,7 @@ export default {
     usageStatistics: false,
     lastVersion: app ? app.getVersion() : 'test',
     autoUpdates: false,
-    language: 'en'
+    language: 'en',
+    email: '',
+    token: ''
```

Comment:
Is it secure to store tokens in the config? I'm not certain whether this is a problem but it seems like very weak protection.